### ISSUE TITLE
Silhouette Method — Better than Elbow Method to find Optimal Clusters

### ISSUE THREAD

**Issue by rhiriyappa** [rhiriyappa/data-science-articles#9]
##TL;DR

Deep dive analysis of Silhouette Method to find optimal clusters in k-Means clustering

##Article Link

**Silhouette analysis** vs **elbow method**
https://towardsdatascience.com/silhouette-method-better-than-elbow-method-to-find-optimal-clusters-378d62ff6891

Compare: **silhouette analysis** vs **elbow method** vs **davies bouldin index** 
https://gdcoder.com/silhouette-analysis-vs-elbow-method-vs-davies-bouldin-index-selecting-the-optimal-number-of-clusters-for-kmeans-clustering/

##Key Takeaway
1. The Davies-Bouldin Index evaluates intra-cluster similarity and inter-cluster differences
2. Silhouette score measure the distance between each data point, the centroid of the cluster it was assigned to and the closest centroid belonging to another cluster. 
3. In elbow method, the score is usually calculated as the mean squared distance between each instance and its closest centroid.
